menu/navigation.json

```diff
@@ -1105,79 +1105,114 @@
-                  },
-                  {
-                    "label": "Upgrade a Database",
-                    "slug": "upgrade-a-database"
-                  },
-                  {
-                    "label": "Connect a Database to a Private Network",
-                    "slug": "connect-database-private-network"
-                  },
-                  {
-                    "label": "Delete a Database",
-                    "slug": "delete-a-database"
-                  }
-                ],
-                "label": "How to",
-                "slug": "how-to"
-              }
-            ],
-            "label": "Managed Database",
-            "slug": "database"
           }
         ],
         "label": "Storage",
         "slug": "storage"
       },
+      {
+        "icon": "database",
+        "items": [
+          {
+          "items": [
+            {
+              "label": "Concepts",
+              "slug": "concepts"
+            },
+            {
+              "label": "Quickstart",
+              "slug": "quickstart"
+            },
+            {
+              "items": [
+                {
+                  "label": "Create a Database",
+                  "slug": "create-a-database"
+                },
+                {
+                  "label": "Add users",
+                  "slug": "add-users"
+                },
+                {
+                  "label": "Manage permissions",
+                  "slug": "manage-permissions"
+                },
+                {
+                  "label": "Manage allowed IPs",
+                  "slug": "manage-allowed-ip-addresses"
+                },
+                {
+                  "label": "Change the volume type",
+                  "slug": "change-volume-type"
+                },
+                {
+                  "label": "Enable Autobackup",
+                  "slug": "enable-autobackup"
+                },
+                {
+                  "label": "Manage manual backups",
+                  "slug": "manage-manual-backups"
+                },
+                {
+                  "label": "Manage Snapshots",
+                  "slug": "manage-snapshots"
+                },
+                {
+                  "label": "Configure advanced settings",
+                  "slug": "configure-advanced-settings"
+                },
+                {
+                  "label": "Upgrade a Database",
+                  "slug": "upgrade-a-database"
+                },
+                {
+                  "label": "Connect a Database to a Private Network",
+                  "slug": "connect-database-private-network"
+                },
+                {
+                  "label": "Delete a Database",
+                  "slug": "delete-a-database"
+                }
+              ],
+              "label": "How to",
+              "slug": "how-to"
+            }
+          ],
+          "label": "For PostgreSQL and MySQL",
```

Comment:
```suggestion
          "label": "Databases for PostgreSQL and MySQL",
```

Reply:
Salut Nour,
Dans la documentation on va laisser "for...", c'est une question d'espace dans le menu.